### ISSUE TITLE
hyStrath.git branch OF-v2112 not present

### ISSUE THREAD

**Issue by karamiag** [vincentcasseau/hyStrath#140]
I tried to install the dev version but when using the command

git clone https://github.com/hystrath/hyStrath.git --branch OF-v2112 --single-branch && cd hyStrath/

I receive an error message 
warning: Could not find remote branch OF-v2112 to clone.
fatal: Remote branch OF-v2112 not found in upstream origin

![Immagine 2023-06-07 161851](https://github.com/hystrath/hyStrath/assets/135842266/21e8fd73-fe42-4c08-9b14-77e3601ba03f)

**Comment by xlxs4** [comment]
It would be cool to have some updates on the current status, as getting v1706 to work on ARM is a pain

**Comment by jarrad301** [comment]
I agree. Any updates on whether the v2112 branch will be ready to release soon?

**Comment by xlxs4** [comment]
I had made https://github.com/xlxs4/aocs-models/tree/main/dsmc (see [Dockerfile](https://github.com/xlxs4/aocs-models/blob/main/dsmc/Dockerfile)) back in the day, which worked just fine, in case it's of any help

**Comment by MadanBK1** [comment]
> I had made https://github.com/xlxs4/aocs-models/tree/main/dsmc (see [Dockerfile](https://github.com/xlxs4/aocs-models/blob/main/dsmc/Dockerfile)) back in the day, which worked just fine, in case it's of any help

do you have downloaded source code by any chance?

**Comment by xlxs4** [comment]
> do you have downloaded source code by any chance?

of what, hyStrath?

I use
- https://sourceforge.net/projects/openfoam/files/v1706/OpenFOAM-v1706.tgz
- https://sourceforge.net/projects/openfoam/files/v1706/ThirdParty-v1706.tgz

to build OpenFOAM, and then just clone the hyStrath repo:

```dockerfile
# ...
RUN wget 'https://sourceforge.net/projects/openfoam/files/v1706/OpenFOAM-v1706.tgz' && \
tar -xzf OpenFOAM-v1706.tgz && \
rm -rf OpenFOAM-v1706.tgz && \
wget 'https://sourceforge.net/projects/openfoam/files/v1706/ThirdParty-v1706.tgz' && \
tar -xzf ThirdParty-v1706.tgz && \
rm -rf ThirdParty-v1706.tgz

# ...
COPY install.sh ${WM_PROJECT_USER_DIR}
RUN git clone https://github.com/hystrath/hyStrath.git --branch master --single-branch && \
mv -f install.sh hyStrath/install.sh

# ...
```

from [Dockerfile](https://github.com/xlxs4/aocs-models/blob/main/dsmc/Dockerfile). The installation script is [lightly modified](https://github.com/xlxs4/aocs-models/blob/main/dsmc/install.sh) because the original hyStrath one did not work
